package.json

```diff
@@ -4,17 +4,36 @@
   "description": "React Context that progressively enhances components",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist/",
+    "yarn.lock"
+  ],
   "dependencies": {
-    "react": "^16.4.0",
-    "@types/react": "^16.3.17"
+    "@types/react": "^16.3.17",
+    "react": "^16.4.0"
   },
   "devDependencies": {
+    "@types/chai": "^4.1.3",
+    "@types/enzyme": "^3.1.10",
+    "@types/mocha": "^5.2.1",
+    "@types/react-dom": "^16.0.6",
+    "chai": "^4.1.2",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-utils": "^1.3.0",
+    "jsdom": "^11.11.0",
+    "jsdom-global": "^3.0.2",
+    "mocha": "^5.2.0",
+    "react-dom": "^16.4.0",
+    "react-reconciler": "^0.11.0",
+    "react-test-renderer": "^16.4.0",
+    "ts-node": "^6.1.0",
     "typescript": "^2.9.1"
   },
   "scripts": {
-    "compile": "tsc",
-    "prepublishOnly": "npm run compile",
-    "test": "echo \"Error: no test specified\" && exit 0"
+    "test:mocha": "cd tests && mocha -r ts-node/register -r jsdom-global/register -r ./setup-enzyme.js ./index.tsx",
+    "compile": "tsc -p src/tsconfig.json",
```

Comment:
(nit) I prefer to use longhand flag syntax for readability. Most of the time I can never remember what the letters stand for!